server/docker-base/Dockerfile

```diff
@@ -448,16 +393,15 @@ RUN \
 # fix permissions
     chmod +r -R ${ANDROID_ROOT} && \
     chmod +w -R ${ANDROID_SDK_ROOT} && \
-    chmod -R 755 ${ANDROID_ROOT}/android-ndk-r${ANDROID_NDK20_VERSION} && \
     chmod -R 755 ${ANDROID_ROOT}/android-ndk-r${ANDROID_NDK25_VERSION} && \
 # check that dx installed properly
-    ls -la ${ANDROID_SDK_BUILD_TOOLS_PATH}/d8 && \
-    which armv7a-linux-androideabi${ANDROID_NDK20_API_VERSION}-clang++ && \
-    which aarch64-linux-android${ANDROID_64_NDK20_API_VERSION}-clang++ && \
+    ls -la ${ANDROID_SDK_BUILD_TOOLS_PATH_33}/d8 && \
     which ${ANDROID_NDK25_BIN_PATH}/armv7a-linux-androideabi${ANDROID_NDK25_API_VERSION}-clang++ && \
-    which ${ANDROID_NDK25_BIN_PATH}/aarch64-linux-android${ANDROID_64_NDK25_API_VERSION}-clang++ && \
+    which ${ANDROID_NDK25_BIN_PATH}/aarch64-linux-android${ANDROID_64_NDK25_API_VERSION}-clang++
 # check that aapt installed correctly
-    aapt v
+# We specify ANDROID_SDK_BUILD_TOOLS_PATH_* in build_input.yml by setting it the first in PATH
+# We can't check aapt in initial setup anymore
+#   aapt v
```

Comment:
to be able to use `aapt` from different SDKs we have to specify them from `build_input.yml`